airbyte-cdk/java/airbyte-cdk/dependencies/src/main/kotlin/io/airbyte/commons/json/Jsons.kt

```diff
@@ -443,7 +445,14 @@ object Jsons {
      * sensitive information in the exception message (which would be exposed with eg,
      * ExceptionUtils.getStackTrace(t).)
      */
-    fun obfuscateDeserializationException(throwable: Throwable): String {
+    fun obfuscateDeserializationException(
+        throwable: Throwable,
+        obfuscate: Boolean = !EnvVariableFeatureFlags().logInvalidJsonDeserialization(),
```

Comment:
NIT: let's avoid the double negative.
`allowJsonInLogs: Boolean=EnvVariable...`

Reply:
good call, done (I went with `showFullExceptionMessage` for the param name)